### ISSUE TITLE
[EKS] Support Inferentia/Neuron Runtime

### ISSUE THREAD

**Issue by samjo-nyang** [bottlerocket-os/bottlerocket#1995]
**What I'd like:**
I think it requires the neuron driver on https://github.com/aws/aws-neuron-sdk

**Any alternatives you've considered:**
Nothing

**Comment by cbgbt** [comment]
Thanks for raising this. We're interested in integrating with Neuron, and it's something we're planning to look into down the road!

**Comment by cbgbt** [comment]
Re-titled this to be consistent with #1075, which is similar but for an ECS Inferentia variant.

**Comment by stmcginnis** [comment]
Is this still needed?

**Comment by samjo-nyang** [comment]
Yes, we are using more neuron instances than I created the ticket. (actively migrating workloads from gpu to neuron)

**Comment by hustshawn** [comment]
Container SSA check-in. IHAC is running ML workloads with Inferentia on EKS. They are quite interested in Bottlerocket in terms of awesome security benefits they get with less overhead. They really want to align the company standards to use Bottlerocket for general business application as well as ML workloads. But the lack of support for Inferentia would affect their adoption.

**Comment by heichow** [comment]
IHAC who is running Stable Diffusion on EKS Inf2, and they wish to adopt [Bottlerocket image cache solution](https://github.com/yubingjiaocn/bottlerocket-images-cache) to reduce the large image (10+GB) pulling time from ECR around 3-4 minutes. Foreseeing the increasing GenAI model hosting with Inferentia, supporting Inferentia/Neuron runtime will have a big impact.
